CHANGELOG.md

```diff
@@ -4,6 +4,28 @@
 
 <!-- changelog start -->
 
+## [0.16.0 (2024-06-06)]
+
+
+### Breaking Changes & Upgrade Guide
+
+- The deprecated coalesce paths (i.e. `(field1|field2)`) feature is now removed. (https://github.com/vectordotdev/vrl/pull/836)
+
+### New Features
+
+- Added experimental `dns_lookup` function. It should be used with caution, since it involves network
+  calls and is therefore very slow.
+
+  authors: esensar (https://github.com/vectordotdev/vrl/pull/764)
```

Comment:
Should this line be here? cc/ @neuronull

Reply:
FYI I followed up with a manual removal of these for now.